### ISSUE TITLE
Sunkiller unit update 

### ISSUE THREAD

**Issue by Mayegelt** [BSData/horus-heresy#2053]
Update to precision fire rule

Old wording
"An enemy model that suffers a Wound, Penetrating Hit or Glancing Hit inflicted by a Shooting Attack, made by a model with this special rule that has not moved in the preceding Movement phase, may not take cover saves."

New wording
"Models may not choose to make a Cover Save, but may still make applicable Armour or Invulnerable saves, against Wounds, Glancing Hits or Penetrating Hits inflicted by attacks made with this special rule. Attacks made as Snap Shots gain no benefit from this special rule, and saves are taken against them as per the standard rules."

https://www.warhammer-community.com/wp-content/uploads/2022/03/H6ygklXe9Fv2FwRe.pdf

**Comment by LachrimallusExterminator** [comment]
Hello,

I currently can’t find the unit. Has it been removed for maintenance? Also, please allow Mournival lists to use the official version as well as the fan-made entry.

Thanks

**Comment by Mayegelt** [comment]
The new official EC only are in heavy support (see picture)

A lot of the battle manual stuff still isn't added. This includes the consuls and various units.
I think I managed to add the Agents of Imperium/emperor/warmaster and not much more.
Again me not being well hasn't helped, and I managed to get a good burst of stuff done a couple of months ago. But since then only really had it in me to focus on official new units, and even then it hasn't been easy at times.

Thank you for the report though. They give me bits to look at and work on if I feel ok.

![Screenshot_2022-03-28-23-46-06-237_net battlescribe mobile rostereditor](https://user-images.githubusercontent.com/20280410/160499731-895a4d37-925b-49e2-b00b-3689557d8a99.jpg)
